src/protocol/template.rs

```diff
@@ -7,12 +7,14 @@
 
 macro_rules! protocol_struct {
     (
+
```

Comment:
Was this newline intentional? It doesn't look intentional.

Reply:
It is not.